mibs/fiberhome/WRI-CPU-MIB.mib

```diff
@@ -0,0 +1,225 @@
+WRI-CPU-MIB DEFINITIONS ::= BEGIN

```

Comment:
Please rename this mib file by dropping the `.mib`. Same for any other mibs.

Reply:
Not sure whether renaming the vendor MIB pays off in the long run. If there is an update from the vendor in future, need to always go through this renaming exercise again. The original MIB names are also declared inside the MIBs. But of cause it looks nicer not to have xx-MIB.MIB or small letters. Will rename for now to follow your standard.